app/views/volunteer_ops/popup/_mixed.html.erb

```diff
@@ -1,11 +1,12 @@
 <button class='close infowindow_close' type='button'>×</button>
 <div class="long-vol-op">
-  <% vol_ops.each do |volop| %>
-    <% if volop.source == 'local' %>
-      <%= render partial: 'volunteer_ops/popup/vol_local', locals: {volop: volop} %>
-    <% elsif volop.source == 'doit' %>
-      <%= render partial: 'volunteer_ops/popup/vol_doit', locals: {volop: volop} %>
+  <ul>
+    <% vol_ops.each do |volop| %>
+      <% if volop.source == 'local' %>
+        <%= render partial: 'volunteer_ops/popup/vol_local', locals: {volop: volop} %>
```

Comment:
here rather than the if/else we could construct the partial name from the volop.source, e.g.  render partial: "volunteer_ops/popup/vol_#{volop.source}"